src/modules/submit/ReviewForm/ReviewFormSection.tsx

```diff
@@ -71,6 +75,7 @@ export const ReviewFormSection = ({
             <Button
               variant="secondary"
               type="button"
+              className={button()}
```

Comment:
question: can we pass the `isResponsive` prop here instead of using a `text-sm` class name?

Reply:
oh, i saw that according to the figma, the button is font size 14px for both the mobile and web view, which is why i created a new class there
![image](https://github.com/user-attachments/assets/c6224970-e27a-415d-818b-38c9e79224d9)

Reply:
but yeah, i think it'll probably look btr if the font was smaller in the mobile view?

Reply:
oh i see, alright then, resolving this comment